shanoir-ng-datasets/src/main/java/org/shanoir/ng/configuration/amqp/RabbitMQDatasetsService.java

```diff
@@ -490,20 +504,23 @@ public String copyDatasetsToStudy(final String data) {
 					res = "Copy worked !";
 				}
 			}
-			LOG.warn("Copy of all datasets done");
 
-			event.setMessage("Copy of dataset over.");
+			event.setMessage("Copy of datasets successful in study [" + studyId + "].");
 			event.setStatus(ShanoirEvent.SUCCESS);
 			event.setProgress(1.0f);
+
 			eventService.publishEvent(event);
+
+			solrService.indexAll();

```

Comment:
IndexAll may be overkill,  if you have the list of created IDs, you can use indexDatasets
(indexAll destroy all the solr index and recreates it)